app/models/filter.rb

```diff
@@ -201,7 +201,14 @@ def parenthesize(string)
   # if we have 0 types, empty validation will set error, we can't have more than one type
   def same_resource_type_permissions
     types = self.permissions.map(&:resource_type).uniq
-    errors.add(:permissions, _('must be of same resource type (%s)') % types.join(',')) if types.size > 1
+    errors.add(
```

Comment:
Absolutely, we have so many of these useless error messages in our codebase. :+1: